_includes/ogp.html

```diff
@@ -1,18 +1,22 @@
 {% capture header %}
 
 {% if page.title %}
-<meta property="og:type" content="article" />
+<meta property="og:type" content="website" />
 <meta property="og:title" content="{{ page.title }}" />
 {% else %}
 <meta property="og:type" content="website" />
+<meta property="og:title" content="Scala.js" />
 {% endif %}
 
-{% include meta.html property="og:site_name" content=site.title %}
+{% if page.description %}
+<meta property="og:description" content="{{ page.description }}" />
+{% else %}
+<meta property="og:description" content="{{ site.description }}" />
+{% endif %}
 
-{% capture page_url %}{{ page.url | replace:'/index.html','/' | prepend: site.url }}{% endcapture %}
-{% include meta.html property="og:url" content=page_url %}
+{% include meta.html property="og:site_name" content=site.title %}
 
-{% capture image_url %}{{ site.url | append: '/assets/img/scala-js-logo-256.png'}}{% endcapture %}
+{% capture image_url %}{{ site.url | append: '/assets/img/scala-js-logo-128.png'}}{% endcapture %}
 {% include meta.html content=image_url property="og:image" %}
```

Comment:
Ah, those pesky demands :D I'll change it back.. it just looked so big in my tests :)